### ISSUE TITLE
Does not go full width at 1440p

### ISSUE THREAD

**Issue by Strykar** [miguelmota/cointop#129]
At `2560x1440` resolution in a windowed (Alacritty) and full screen (Kitty) terminal, cointop does not extend fully to the right.
See https://i.imgur.com/B5U210U.png

**Comment by miguelmota** [comment]
Hi @Strykar, would you expect all the columns to be evenly spaced out in this case?

**Comment by Strykar** [comment]
@miguelmota Hi, yes, auto adjusting the columns based on width would be nice, esp. at higher resolutions.

**Comment by lyricnz** [comment]
Is it the chart you're interested in, or the table?  (the chart is easy/trivial, the table code would be a lot more involved)

**Comment by Strykar** [comment]
> Is it the chart you're interested in, or the table? (the chart is easy/trivial, the table code would be a lot more involved)
> 
No, that appears like the current chart with just the graph and date columns spilling over to the right.

**Comment by simon-anz** [comment]
I asked: Is it the chart you're interested in, or the table?   (and I provided a solution for the chart).

It seems you're interested in the table too?  What do you suggest - just adding padding to all the columns evenly until it fills the space?  This may make it hard to read / line-up.

**Comment by lyricnz** [comment]
I asked: Is it the chart you're interested in, or the table? (and I provided a solution for the chart).

It seems you're interested in the table too? What do you suggest - just adding padding to all the columns evenly until it fills the space? This may make it hard to read / line-up?

**Comment by Strykar** [comment]
> I asked: Is it the chart you're interested in, or the table? (and I provided a solution for the chart).
> 
> It seems you're interested in the table too? What do you suggest - just adding padding to all the columns evenly until it fills the space? This may make it hard to read / line-up?

Both, chart and table.
Honestly I can't think of a better solution than padding, which may not be worth the effort as you point out.

**Comment by deathtrip** [comment]
I would like to add that it happens for me on 1080p also, so it's probably independent of  resolution.
I also think padding is the best solution, as right now the columns seem squeezed together if the values are very large.
Some coin names are even cut off, which doesn't make sense when there's so much space left.
